### ISSUE TITLE
Links in the Readme lead to a 404

### ISSUE THREAD

**Issue by thapapratyush** [PolymerElements/iron-icons#66]
The attached links in the last two lines of the Readme leads to a 404 error. 

These are the lines:
See _iron-icon_ for more information about working with icons.

See _iron-iconset_ and _iron-iconset-svg_ for more information about how to create a custom iconset.

where, _iron-icon_, _iron-iconset_ and _iron-iconset-svg_ lead to a webpage not found on Github.

Browser and OS used:
Google Chrome on OSX.

**Comment by notwaldorf** [comment]
Yeah, the links are made so that they work in the elements catalog; this unfortunately means they're a bit broken in the README.

/cc @samuelli, since I think this might be a thing to consider for the new catalog

**Comment by samuelli** [comment]
Hmm, we don't do anything special here. For now, my preference is less magical, so the new catalog behaves the same as GitHub.

My personal preference is probably for fully-qualified URLs here, since we don't plan on hosting documentation on GitHub anymore.

**Comment by steren** [comment]
There are 2 pending PRs which are fixing this: https://github.com/PolymerElements/iron-icons/pull/73 and https://github.com/PolymerElements/iron-icons/pull/58

Could on of them be merged?
